sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgBlockBuilder.cs

```diff
@@ -42,7 +42,7 @@ public BasicBlock CreateBasicBlock(Block block)
             var ucfgBlock = CreateBlockWithId(blockIdProvider.Get(block));
 
             ucfgBlock.Instructions.AddRange(
-                block.Instructions.Select(instructionBuilder.Create).WhereNotNull());
+                block.Instructions.SelectMany(instructionBuilder.Create).WhereNotNull());
```

Comment:
I think this `WhereNotNull` should not be needed, as Create would return Enumerable.Empty when no instruction was created